template/templates/template_example.yml.erb

```
@@ -0,0 +1,12 @@
+each_record:
+  - type: A
+    fqdn: abc.123.com.
+  - type: TXT
+conflict_with:
+  - type: TXT
+    fqdn: <%= "#{record.fqdn}.more.domain.com" %>
+injected_record:
+- type: CNAME
+  ttl: 3600
+  fqdn: <%= "#{record.fqdn}.added.information.com." %>
```

Would work too: (nitpicking)

```suggestion
  fqdn: <%= record.fqdn %>.added.information.com.
```